charts/cray-etcd-test/Chart.yaml

```diff
@@ -28,7 +28,7 @@ dependencies:
   version: "~10.0.0"
   repository: https://artifactory.algol60.net/artifactory/csm-helm-charts
 - name: cray-etcd-base
-  version: "~1.1.1"
+  version: "1.2.0-20240516154242+ff2fc0e"
```

Comment:
This will need to be changed to a released version of `cray-etcd-base`, probably another PR after this one is merged and a new release is created?

Reply:
Yes, I will do a follow up PR once this change has merged.

Reply:
Okays, thanks